test/Singleton/Phoenix.cpp

```diff
@@ -81,6 +81,9 @@ class Example
 
 int main(int argc, char* argv[])
 {
+    (void)argc;
```

Comment:
you can also use UNREFERENCED_PARAMETER() in Windows or comment the funcion parameters just leaving types